### PR TITLE
New `setnick` command

### PR DESCRIPTION
**Pull request by DTS-11**
Changes nickname of a user in the server.

### CODE REVIEW

**Review by sifte — CHANGES_REQUESTED** [review]
there a conflicts that need to be resolved

**Comment by Yat-o** [comment]
Did you not pull or something before you did this?

File: src/cogs/mod.py [hunk 1]
```diff
@@ -12,6 +12,17 @@ class Mod(commands.Cog):
...
+    @commands.command()
+    @commands.guild_only()
+    @commands.has_permissions(manage_nicknames=True)
+    @commands.bot_has_permissions(manage_nicknames=True)
+    async def setnick(self, ctx, member: discord.Member, *, nick):
+        """
+        Set a custom nick-name.
+        """
+        await member.edit(nick=nick)
+        await ctx.send(f'Nickname for {member.name} was changed to {member.mention}')
```
Comment (sarthak-py) [inline src/cogs/mod.py:25]: misses hierarchy check, no handling if the bot is unable to edit the nickname of the member ,

**Review by sarthak-py — CHANGES_REQUESTED** [review]

File: src/cogs/mod.py (hunk 1 above)
Comment (DTS-11) [reply src/cogs/mod.py:25]: we can make error handling for that in another cog...

Comment (sarthak-py) [reply src/cogs/mod.py:25]: i dont see how the error handler will handle a hierarchy check and discord.Forbidden Error for a single command

Comment (DTS-11) [reply src/cogs/mod.py]: I will make another PR. closing this now

**Comment by DTS-11** [comment]
I'll make another PR... idk much of github.. (this is so complicated :/)

**Pull request closed without merging**